embedded_fab/content.dart

```diff
@@ -0,0 +1,415 @@
+// Copyright 2018-present the Material Components for Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+List<Widget> get navigationViews => [
+      NavigationView(
+        icon: Icon(Icons.inbox),
+        title: 'Inbox',
+        order: [0, 5, 1, 4, 3, 2, 6, 7],
+        times: [
+          '2:44pm',
+          '2:55pm',
+          '3:08pm',
+          'Nov 2',
+          'Nov 2',
+          'Nov 4',
+          'Nov 4',
+          'Nov 6'
+        ],
+      ),
+      NavigationView(
+        icon: Icon(Icons.star),
+        title: 'Starred',
+        order: [5, 2, 0, 6, 1, 4, 3, 7],
+        times: [
+          '2:08pm',
+          '2:08pm',
+          '2:11pm',
+          'Nov 3',
+          'Nov 4',
+          'Nov 5',
+          'Nov 5',
+          'Nov 6'
+        ],
+      ),
+      NavigationView(
+        icon: Icon(Icons.send),
+        title: 'Sent',
+        order: [1, 4, 6, 2, 5, 3, 7, 0],
+        times: [
+          '12:08pm',
+          '1:07pm',
+          '2:08pm',
+          '3:08pm',
+          '4:08pm',
+          '6:01pm',
+          '6:40pm',
+          '7:20pm'
+        ],
+      ),
+      NavigationView(
+        icon: Icon(Icons.archive),
+        title: 'Archived',
+        order: [6, 7, 2, 0, 4, 1, 3, 5],
+        times: [
+          '2:08pm',
+          '2:09pm',
+          'Nov 2',
+          'Nov 2',
+          'Nov 3',
+          'Nov 6',
+          'Nov 6',
+          'Nov 8'
+        ],
+      ),
+    ];
+
+class _MyHomePageState extends State<MyHomePage> {
+  int _selectedIndex = 0;
+  List<Widget> _navigationViews = [
```

Comment:
These aren't needed anymore because of the getter above right?

Reply:
Yeah. Accidentally included.